src/Google.Api.Gax/Api/Gax/MetadataExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Grpc.Core;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Google.Api.Gax
+{
+    internal static class MetadataExtensions
+    {
+        public static void Add(this Metadata metadata, IEnumerable<Metadata.Entry> items)
```

Comment:
Side piece of trivia: this is the first time I've seen the C# 6 feature of having extension methods used by collection initializers. And I forgot to mention it in my recent talk :(